src/modules/core/database/prisma/prisma.service.ts

```diff
@@ -111,4 +117,31 @@ export class PrismaService
     }
     return;
   }
+
+  /**
+   * 매핑테이블의 매핑된 데이터 정보 유효성 검사를 해주는 메서드
+   */
+  async validateMappedDataOrFail<T extends LessonHashtagMapping | Major>(
```

Comment:
```suggestion
  async validateMappedDataOrFail<T extends LessonHashtagMapping>(
```
이거 Major 는 매핑 테이블은 아니고 예시를 위해 아무거나 하나 추가한거예요 ㅎㅎ